add v1.10.2 release download page (#554)
klaytn/klaytn-docs: docs/installation-guide/download/v1.10.2.md

- [kpn-baobab-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/kpn-baobab-v1.10.2-0-darwin-10.10-amd64.tar.gz)
- [ken-baobab-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/ken-baobab-v1.10.2-0-darwin-10.10-amd64.tar.gz)

Common Packages
- [kscn-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/kscn-v1.10.2-0-darwin-10.10-amd64.tar.gz)
- [kspn-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/kspn-v1.10.2-0-darwin-10.10-amd64.tar.gz)
- [ksen-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/ksen-v1.10.2-0-darwin-10.10-amd64.tar.gz)
- [kbn-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/kbn-v1.10.2-0-darwin-10.10-amd64.tar.gz)
- [kgen-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/kgen-v1.10.2-0-darwin-10.10-amd64.tar.gz)
- [homi-v1.10.2-0-darwin-10.10-amd64.tar.gz](https://packages.klaytn.net/klaytn/v1.10.2/homi-v1.10.2-0-darwin-10.10-amd64.tar.gz)

## Windows <a id="windows"></a>

- Not supported yet

# Release Notes <a id="release-notes"></a>

> Klaytn v1.10.2 contains a hardfork upgrade that results in backward-incompatible changes. All Baobab/Cypress nodes would thus have to be upgraded to v1.10.2 or higher before the target block number. The hardfork implements [[KIP-103](https://github.com/klaytn/kips/pull/104)] that is a technical specification of [[KGP-6](https://govforum.klaytn.foundation/t/kgp-6-proposal-to-establish-a-sustainable-and-verifiable-klay-token-economy/157)].

Reviewer: ```suggestion
Klaytn v1.10.2 contains a hardfork upgrade that results in backward-incompatible changes. All Baobab/Cypress nodes would thus have to be upgraded to v1.10.2 or higher before the target block number. The hardfork implements [[KIP-103](https://kips.klaytn.foundation/KIPs/kip-103)] that is a technical specification of [[KGP-6](https://govforum.klaytn.foundation/t/kgp-6-proposal-to-establish-a-sustainable-and-verifiable-klay-token-economy/157)].
```

Author: thx!

The hardfork schedule is as follows:

**Baobab Testnet**
- BlockNumber: 119145600 (estimated time: 6 Apr, 4:20 AM UTC+9)
- Hardfork contents: KIP-103 hardfork (Baobab already applied Kore hardfork)
- TreasuryRebalance contract address: 0xD5ad6D61Dd87EdabE2332607C328f5cc96aeCB95

**Cypress Mainnet**
- BlockNumber: 119750400 (estimated time: 17 Apr, 01:01 AM UTC+9)
- Hardfork contents: Kore, KIP-103 hardforks 
- TreasuryRebalance contract address: 0xD5ad6D61Dd87EdabE2332607C328f5cc96aeCB95

> *You can find detailed information on TreasuryRebalance contract from [KIP-103](https://github.com/klaytn/kips/pull/104)

Reviewer: ```suggestion
*You can find detailed information on TreasuryRebalance contract from [KIP-103](https://kips.klaytn.foundation/KIPs/kip-103)
```

NOTE: At the activation of [KIP-103](https://github.com/klaytn/kips/pull/104), the balance of the Treasury account will be changed in an unusual way. You can find the change result on the TreasuryRebalance contract.
NOTE: Some APIs (`governance_getStakingInfo`, `klay_getStakingInfo`, `governance_getRewards`) change their returning field names of `KGF`, `KIR` to `kff`, `kcf` respectively. The old field names will remain before v1.11.0 for backward compatibility.

## Hardfork features 

> - [KIP-103](https://github.com/klaytn/kips/pull/104): Implementing TreasuryRebalance (#1803 #1809 #1810)

Reviewer: ```suggestion
- [KIP-103](https://kips.klaytn.foundation/KIPs/kip-103): Implementing TreasuryRebalance (#1803 #1809 #1810)
```
